src/Microsoft.DotNet.Cli.Utils/AnsiConsole.cs

```diff
@@ -63,7 +63,7 @@ public void WriteLine(string message)
             var escapeScan = 0;
             for (;;)
             {
-                var escapeIndex = message.IndexOf("\x1b[", escapeScan);
+                var escapeIndex = message.IndexOf("\x1B[", escapeScan);
```

Comment:
We should make this (and all the other IndexOfs) case-insensitive since it's checking a Hex number, so the casing can be mixed.